libs/types/src/system_settings.ts

```diff
@@ -148,12 +148,6 @@ export const DEFAULT_MARK_THRESHOLDS: Readonly<MarkThresholds> = {
   writeInTextArea: 0.05,
 };
 
-/**
- * The default bitonal threshold for scanning ballots.
- * See Section 2.1.43 of the PDI PageScan software specification.
- */
-export const DEFAULT_BITONAL_THRESHOLD = 75;
```

Comment:
Still not completely removed from lower levels in the stack but continued the removal while I was passing through (reminder that this is no longer needed because we now scan in grayscale rather than B&W)

Reply:
Thanks for continuing this removal! Should we remove lower as well? (not necessarily in this PR)

Reply:
Yeah I think we should! We have this tech-improvement issue open until we complete that: https://github.com/votingworks/vxsuite/issues/6672